context/src/junit/bindings.rs

```diff
@@ -730,3 +903,117 @@ impl BindingsJunitReportValidation {
             .count()
     }
 }
+
+#[cfg(feature = "bindings")]
+#[test]
+fn parse_test_report_to_bindings() {
+    use prost_wkt_types::Timestamp;
+    let mut test1 = TestCaseRun::default();
+    test1.id = "test_id1".into();
+    test1.name = "test_name".into();
+    test1.classname = "test_classname".into();
+    test1.file = "test_file".into();
+    test1.parent_name = "test_parent_name1".into();
+    test1.line = 1;
+    test1.status = TestCaseRunStatus::Success.into();
+    test1.attempt_number = 1;
```

Comment:
optional: It might save you some characters to use the partial spread. Same goes for other instances of this

```suggestion
    let mut test1 = TestCaseRun {
        id: "test_id1".into(),
        name: "test_name".into(),
        classname: "test_classname".into(),
        file: "test_file".into(),
        parent_name: "test_parent_name1".into(),
        line: 1,
        status: TestCaseRunStatus::Success.into(),
        attempt_number: 1,
        ..Default::default()
    };
```